---
title: "Microbial species definitions and horizontal gene transfer"
author: "Julia Pierce (24366429)"
date: "4/22/2018"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
Since the discovery of evolution, we have been constantly updating and rethinking how life came about and developed into the distinct animals and microbes we see today. Much of this done through the grouping of creatures based on shared properties, and tracking the divergences in development that lead to these distinct groups (mollusks vs. anthropods, hemichordata vs. chordata) using phylogenetic trees. However, when this is attampted with microbes at a high resolution, ie. sat the level of a species, things get complicated. Microbes have no one set way of adapting to the environment, and have the ability to share genetic material to impart new heterologus functions. This sharing of material blurs the lines between microbial species, and has other far-reaching implications. So how do we go about defining a microbial species? How does this sharing of genetic material between microorganisms shape our world? Is a clear definition of microbial species even relevant? 

As new tchnologies emerge, new methods of species identification open to us and allow for new species definitions. The current species definition is based on 97% (or more) identity of the 16S rRNA, or more than 70% DNA-DNA hybridization (1). When defining speceis of multicellular organisms, there are three methods of categorization: typological, based on shared traits, i.e. morphology; biological, based on breeding habits, and; evolutionary, based on lineage) (2). As microbes share the same 'breeding' habits (ie. cell division) and morphology isn't sufficient for microbial distinction, the only option is to make inferences based on lineage and simialrites in genetic material (3). However, in analyzing the microbial genome sequences, diversity often forms a continuum rather than adhereing to strict boundaries (3). This is due to the effect of horizontal gene transfer.

Horizontal gene transfer (HGT) is the sharing of genetic material between two non-phylogenetically linked cells; ie, not between a daughter and parent cell. HGT is responsible for haring phenotypic traits, such aas antibiotic resistance, a certain metabolic pathway, or virulence (etc.) (3). This makes species definitions based on key diagnostic phenotypes unreliable, as traits can be present in a wide variety of microbes (3). On the other side of things, this can also complication comparisons between different strains of the the same species, as due to HGT these different strains can have very different geneome. For instance, a study comparing three strains of *E. coli* found that only 40% of predicted proteins were shared among all three strains (4). This also implies that functional pathways conferred via HGT depend on environment, making any "one rule fits all" for predicting HGT and its effect of phenotype using current species definition impossible. This implies that a broader species definition is needed, anchored using key phenotypic traits. All of this explains why HGT complicates a genomic species definition, but it still does not explain why HGT occurs nor its prevalence. 

HGT causes widespread functional distribution, resulting in many different species expressing the same funtion, and most species as a whole expressing all of the major metabolic pathways (5). This capability to express multiple metabolisms confers the abilities necessary for species to survive through environmental upheaval, and contributes to environmental resilience (6). One such example is the ubiquitos protein proteorhodopsin, a light-activated system responsible for ATP production via the flux of protons across the cell membrane. This protein is compact, comprising only a few genes, and is thus easy to transfer and unlikely to be dropped from the genome, explaining its widespread distribution (7). These two effects of HGT, widespread metabolic pathways and the evolutionary advantage of genomically retaining said pathways, results in microbes "guarding metabolisms" and ensuring all environmental niches are filled (5). Microbes maintain the biosphere and its many chemical cycles through their metabolic processes, and ensuring all of these functions are still capable of being expressed secures the continued cycling of biogeochemicals (8). HGT is responsible for the continued existence of microbes and their metabolic diversity, and the health of our environment.

All this begs the question, is a clear definition of microbial species even required? Without a clear definition of species, one could analyze two identical samples with differing results. Depending on the field of study, this could have wider implications. In the field of medicine, to determine the correct treatment it is necessary to clearly differentiate between two highly similar microbes, such as *Bacillus anthracis* and *Bacillus cereus* (3). However, this is a case of differentiation between named species, which does not necessarily translate to biologically discrete species. If a set of rules for species definition were to be determined using named species, there is no one "right answer". If you define a species by analyzing a few strains from different environments, your set of rules will be drastically different than if you were to analyze strains from similar environments(3, 1). So I propose that if we can't even get a single species definition based on current methods and findings, then don't try to force a single definition. There should still be rules, but these should be tailored to the field of study. For medical uses, there needs to be a very clear definition, and likely some analysis based on virulence. For other applications in which clear identification does not have potentially hazardous ramification, such as WWTP, analyses and definitions can be done on a looser basis. This could be the continued use of 97% identity of 16S rRNA, it could be a switch to the use of Genus and a clustering method, similar to a study done by Konstantinidis et al. (9). As long as the definition is transparent and defendable, adhering to a single species definition should not be necessary in our current age and technological capabilites.

Horizontal gene transfer is key to the continued functioning of our environment, biogeochemical cycles, and microbial diversity, although it greatly complicates matters when defining microbial species. For those fields, such as medicine, in which proper identification of named-species is highly urgent, a clear species definition is necessary. But for many other fields, given the current technology, a species definition at this time does not seem relevant. Other metrics may be used, replacing species with genus and clustering methods, tailored to the needs of the study. Although alternative methods to a species definition would not fully elucidate divergences in evolution, so much of microbial life exists on a continuum that this is not cause for concern. 

**References** 

1. [Hollricher K. 2007. "Species don't really mean anything in the bacterial world". Lab Times, Microbial systematics. **5**:22-25](http://www.labtimes.org/labtimes/issues/lt2007/lt05/lt_2007_05_22_25.pdf)

2. [Boggs CL. 2001. Species and Speciation. International Encyclopedia of the Social and Behavioral Sciences. Elsevier.](https://www.sciencedirect.com/science/referenceworks/9780080430768)

3. [Konstantinidis K, Kolter R. 2017. Do bacterial species really exist and why should we care?. Ecology, Evolution. Online.](http://schaechter.asmblog.org/schaechter/2017/01/do-bacterial-species-really-exist-and-why-should-we-care.html)

4. [Welch RA, Burland V, Plunkett III G, Redford P, Roesch P, Rasko D, Buckles EL, Liou SR, Boutin A, Hackett J, Stroud D, Mayhew GF, Rose DJ, Zhou S, Schwartz DC, Perna NT, Mobley HLT, Donnenberg MS, Blattner FR. 2002. Extensive mosaic structure revealed by the complete genome sequence of uropathogenic *Escherichia coli*. PNAS. **99**(26):17020-17024.](www.pnas.org cgi doi 10.1073 pnas.252529799)

5. [Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive Earth's biogeochemical cycles. Science. **320**:1034-1039.](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.474.2161&rep=rep1&type=pdf)

6. [Lal S, Cheema S, Kalia VC. 2008. Phylogeny vs genome reshuffling: horizontal gene transfer. Indian J. Microbiol. **48**:228â€“242](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3450171/pdf/12088_2008_Article_34.pdf)

7. [Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF. 2007. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologus host. PNAS. **104**(13):5590-5595.](www.pnas.org cgi doi 10.1073 pnas.0611470104)

8. Madsen E. 2005. Identifying microorganisms responsible for ecologically significant biogeochemical processes. Nature. **3**:439-446.

9. [Konstantinidis K., Ramette A, Tiedje J. The bacterial definition in the genomic era. The Royal Society, Philosophical Transactions. **361**:1929-1940.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1764935/pdf/rstb20061920.pdf)