---
title: "Problem_Set_4"
author: "Julia Pierce (24366429)"
date: "3/5/2018"
output: html_document
---
### Problem set_04 “Fine-scale phylogenetic architecture”
#### Learning objectives: 

* Gain experience estimating diversity within a hypothetical microbial community

#### Part 1: Description and enumeration
Obtain a collection of "microbial" cells from "seawater". The cells were concentrated from different depth intervals by a marine microbiologist travelling along the Line-P transect in the northeast subarctic Pacific Ocean off the coast of Vancouver Island British Columbia. 

Sort out and identify different microbial "species" based on shared properties or traits. Record your data in this Rmarkdown using the example data as a guide.

Once you have defined your binning criteria, separate the cells using the sampling bags provided. These operational taxonomic units (OTUs) will be considered separate "species". This problem set is based on content available at [What is Biodiversity](http://cnx.org/content/m12179/latest/).

For example, load in the packages you will use.
```{r message=FALSE}
library(kableExtra)
library(knitr)
library(tidyverse)
library(vegan)
```

**For the original community**
```{r}
example_data1 = data.frame(
  number = c( 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15),
  name = c("Hershey's", "M&Ms", "Coated Bears", "Classic Bears", "Strings", "Wine Gummies", "Strawberry", "Swirl", "Octopi", "Soda Bottles", "Watermelon", "Jolly Ranchers", "Spheres", "Lego", "Skittles"),
  characteristics = c("chocolate kiss", "candy-covered chocolate", "coated bear soft gummy", "bear soft gummy", "long strings, soft matte gummy", "oblong soft matte gummy", "strawberry sugar-coated gummy", "swirly sugar-coated gummy", "octopi sugar-coated gummy", "bottle sugar-coated gummy", "watermelon sugar-coated gummy", "hard oblong gummy", "hard spherical gummy", "bricks of crunchy sugar", "elliptical crunchy sugar"),
  occurences = c(16, 214, 3, 101, 14, 8, 1, 3, 6, 3, 2, 131, 19, 17, 197)
)

example_data1 %>% 
  kable("html") %>%
  kable_styling(bootstrap_options = "basic", font_size = 10, full_width = F)
```
	
**For my sample of the original community:**
```{r}
example_data1_subsample = data.frame(
  number = c( 1, 2, 3, 4, 5, 6, 7, 8, 9, 10),
  name = c("Hershey's", "M&Ms", "Classic Bears", "Wine Gummies", "Swirl", "Soda Bottles", "Jolly Ranchers", "Spheres", "Lego", "Skittles"),
  characteristics = c("chocolate kiss", "candy-covered chocolate", "bear soft gummy", "oblong soft matte gummy", "swirly sugar-coated gummy", "bottle sugar-coated gummy", "hard oblong gummy", "hard spherical gummy", "bricks of crunchy sugar", "elliptical crunchy sugar"),
  occurences = c(3, 52, 23, 2, 1, 1, 36, 5, 7, 43)
)

example_data1_subsample %>% 
  kable("html") %>%
  kable_styling(bootstrap_options = "basic", font_size = 10, full_width = F)
```

**Is your collection of microbial cells from seawater representative of the actual diversity of microorganisms inhabiting waters along the Line-P transect? Were the majority of different species sampled or were many missed?**
The diversity is much lower than the original sample, with only 2/3 of the original species represented in my community sample. Those species that were missed were either part of the "rare biome" (things like coated bears, strawberry, octopi, and watermelon) while the other missing species was on the rare side and also tangled together into a "biofilm" rather than existing as a more easily separatable cell.
 
 
#### Part 2: Collector's curve
To help answer the questions raised in Part 1, you will conduct a simple but informative analysis that is a standard practice in biodiversity surveys. This analysis involves constructing a collector's curve that plots the cumulative number of species observed along the y-axis and the cumulative number of individuals classified along the x-axis. This curve is an increasing function with a slope that will decrease as more individuals are classified and as fewer species remain to be identified. If sampling stops while the curve is still rapidly increasing then this indicates that sampling is incomplete and many species remain undetected. Alternatively, if the slope of the curve reaches zero (flattens out), sampling is likely more than adequate.

To construct the curve for your samples, choose a cell within the collection at random. This will be your first data point, such that X = 1 and Y = 1. Next, move consistently in any direction to a new cell and record whether it is different from the first. In this step X = 2, but Y may remain 1 or change to 2 if the individual represents a new species. Repeat this process until you have proceeded through all cells in your collection.

**For my sample:**
```{r}
example_data2 = data.frame(
  x = c(1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173),
  y = c(1,2,3,4,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,7,7,7,7,7,7,7,8,8,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10)
)

ggplot(example_data2, aes(x=x, y=y)) +
  geom_point() +
  geom_smooth() +
  labs(x="Cumulative number of individuals classified", y="Cumulative number of species observed")
```

**Does the curve flatten out? If so, after how many individual cells have been collected?**
The curve roughly flattens out after ~82 individual cells have been collected. This is roughly half of the total number of cells (173).

**What can you conclude from the shape of your collector's curve as to your depth of sampling?**
As my collector's curve becomes roughly flat, its safe to say that the sampling depth was adaquate, so that the sample was representative of the original community.

#### Part 3: Diversity estimates (alpha diversity)
Using the table from Part 1, calculate species diversity using the following indices or metrics.

##### Diversity: Simpson Reciprocal Index

$\frac{1}{D}$ where $D = \sum p_i^2$

$p_i$ = the fractional abundance of the $i^{th}$ species

**For the original community**
```{r}
species1 = (16/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species2 = (214/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species3 = (3/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species4 = (101/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species5 = (14/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species6 = (8/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species7 = (1/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species8 = (3/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species9 = (6/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species10 = (3/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species11 = (2/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species12 = (131/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species13 = (19/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species14 = (17/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))
species15 = (197/(16+214+3+101+14+8+1+3+6+3+2+131+19+17+197))

1 / ((species1^2) + (species2^2) + (species3^2) + (species4^2) + (species5^2) + (species6^2) + (species7^2) + (species8^2) + (species9^2) + (species10^2) + (species11^2) + (species12^2) + (species13^2) + (species14^2) + (species15^2))
```

**For my sample of the original community**
```{r}
#For my subsample
species01 = (5/(173))
species02 = (7/(173))
species03 = (1/(173))
species04 = (3/(173))
species05 = (1/(173))
species06 = (2/(173))
species07 = (23/(173))
species08 = (36/(173))
species09 = (43/(173))
species010 = (52/(173))

1 / ((species01^2) + (species02^2) + (species03^2) + (species04^2) + (species05^2) + (species06^2) + (species07^2) + (species08^2) + (species09^2) + (species010^2))
```

The higher the value is, the greater the diversity. The maximum value is the number of species in the sample, which occurs when all species contain an equal number of individuals. Because the index reflects the number of species present (richness) and the relative proportions of each species with a community (evenness), this metric is a diveristy metric. Consider that a community can have the same number of species (equal richness) but manifest a skewed distribution in the proportion of each species (unequal evenness), which would result in different diveristy values.

**What is the Simpson Reciprocal Index for your sample?**
The Simpson Reciprocal Index for my sample is 4.628, which is a moderate value (maximum value is 10). So the sample was moderately diverse.

**What is the Simpson Reciprocal Index for your original total community?**
The Simpson Reciprocal Index for the original community is 4.772, which is on the low side (maximum value is 15), and very close to the Simpson Reciprocal Index for my sample from this original community. This implies that the diversity of the original community (and therefore my sample from the original community) is dominated by those species found in high quantities - the rare species of which there are few, like Watermelon and Strawberry, do not have much impact on diversity.

##### Richness: Chao1 richness estimator
Another way to calculate diversity is to estimate the number of species that are present in a sample based on the empirical data to give an upper boundary of the richness of a sample. Here, we use the Chao1 richness estimator. 

$S_{chao1} = S_{obs} + \frac{a^2}{2b})$

$S_{obs}$ = total number of species observed
a = species observed once
b = species observed twice or more

**So the $S_{chao1}$ for my sample of the original community is:**

```{r}
#$S_{obs}$ = 10
#a = 2
#b = 8

10 + ((2^2)/(2*8))
```

**So the $S_{chao1}$ for the original community is:**

```{r}
#$S_{obs}$ = 15
#a = 2
#b = 13

15 + ((2^2)/(2*13))
```

#### Part 4: Alpha-diversity functions in R
We've been doing the above calculations by hand, which is a very good exercise to aid in understanding the math behind these estimates. Not surprisingly, these same calculations can be done with R functions. Since we just have a species table, we will use the `vegan` package. You will need to install this package if you have not done so previously. 

**What are the Simpson Reciprocal Indices for your sample and community using the R function?**

**For the original community**
```{r}
example_data1_diversity = 
  example_data1 %>% 
  select(name, occurences) %>% 
  spread(name, occurences)

example_data1_diversity
#Then, we can calculate the Simpson Reciprocal Index using the `diversity` function.
diversity(example_data1_diversity, index="invsimpson")
```

**For my sample of the original community**
```{r}
example_data2_diversity = 
  example_data1_subsample %>% 
  select(name, occurences) %>% 
  spread(name, occurences)

example_data2_diversity
diversity(example_data2_diversity, index="invsimpson")
```

**What are the chao1 estimates for your sample and community using the R function? Verify that these values match your previous calculations.**
We can calculate the Chao1 richness estimator (and others by default) with the the `specpool` function for extrapolated species richness. This function rounds to the nearest whole number so the value will be slightly different that what you've calculated above.

**For the original community**
```{r}
specpool(example_data1_diversity)
```
**For my sample of the original community**
```{r}
specpool(example_data2_diversity)
```
Given that these values round to the nearest whole number, they match all of my previously calculated values for both the original community and the sample of the original community.

#### Part 5: Concluding activity
If you are stuck on some of these final questions, reading the [Kunin et al. 2010](https://www.ncbi.nlm.nih.gov/pubmed/19725865) and [Lundin et al. 2012](https://www.ncbi.nlm.nih.gov/pubmed/23760801) papers may provide helpful insights. 

**How does the measure of diversity depend on the definition of species in your samples?**
Diversity is directly dependent on the number of species in a sample, and the number of cells belonging to that species. Depending on how you define your species, the number of cells belonging to that species will change, changing your diversity. For example, using my original sample, one group member further broke down species as including the candy color as well as the type (for 52 final species); using this definition, the number of cells in each species will be much closer amoung species (in the range of ~30-1 instead of in the range ~200-1). In this finer definition using 52 species, things are more evenly populus, and the diversity won't be defined by a few large groups (like M&Ms, Jolly Ranchers, Skittles, and Gummy Bears), so the diversity will be larger. Also, diversity is larger when you have more species defined.

**Can you think of alternative ways to cluster or bin your data that might change the observed number of species?**
Yes, I could have further separated things by color. If I wanted to go even deeper, I could have separated colors down based on manufacturing imperfections - partially scratched off m's on M&Ms, lopsided gummy bears, gummy bears with a little bit of the wrong gummy color, etc. If I were to go down to the level of manufacturing imperfections, I would also get in to subjective definitions - do I divide the scrathed off m's by the degree to which they are scratched off? I say this one looks 50% scratched off, and it should go with the 50-60% scratched off yellow M&Ms. But maybe you look at the same 50% scratched off m, and say it only goes with the 50-52% scratched off yellow M&Ms? We could be looking at the same characteristics, but choose different 'cutoff points'. 

**How might different sequencing technologies influence observed diversity in a sample?**
It depends on the section of genetic material you're analyzing, what percent sequence homology is used, different criteria for cleaning up data, etc. Nearly every choice you make can effect your final species definition, which will effect your final diversity. 

**Proposed rules for species definition, discussed with group:**
1) Use an updated, curated database 
2) Proper filtering + cleaning data (get rid of sequencing errors, etc), more stringent with data control
3) Choice of Database should be representative of the data
4) Use the same 16S rRNA region for each organism (and the same primers)
   -There could be a primer bias 
5) Use a consistent protocol for DNA extraction (Sampling method)
   -Different standard method for extracting terrestrial vs. marine environments
6) Define- clearly what is a “species”
   -97% sequence homolgy
7) Don't’ have to classify under the species level
   -Allow more “samples” collapse together
